app/helpers/ui_helper.rb

```diff
@@ -14,6 +14,7 @@ def add_element_button(label, container_selector:, **html_attrs, &block)
               add_dest_selector: container_selector,
               action: "click->form-input#addItem:prevent" },
       role: "button",
+      id: "__add_line_item",
```

Comment:
This assumes we'd never have more than one "add element button" on a page. Can we key off something other than ID?

Reply:
Fair point. I will move the id back to being passed in as an argument.

While we could change it to being an `.add_line_item` class here, we would have to modify the JS that clicks this button automatically to find the element by class rather than id. And then use some logic to determine which (or all) of the add element buttons to click. Which feels like coding for some hypothetical future case of having mutliple add item buttons, which doesn't exist yet.

Reply:
Moved this id back to being an argument.